test/gc-dialects/OneDNNGraph/onednn-graph-to-linalg.mlir

```diff
@@ -0,0 +1,41 @@
+// RUN: gc-opt --split-input-file -pass-pipeline="builtin.module(func.func(convert-onednn-graph-to-linalg))" %s -verify-diagnostics -o -| FileCheck %s
+
+// CHECK-LABEL: @matmul
+func.func @matmul(%arg0: tensor<128x512xbf16>, %arg1: tensor<512x256xbf16>) -> tensor<128x256xbf16> {
+  // CHECK: [[C0:%.+]] = arith.constant 0
+  // CHECK: [[INIT:%.+]] = tensor.empty()
+  // CHECK: [[FILLED:%.+]] = linalg.fill ins([[C0]] : bf16) outs([[INIT]] : tensor<128x256xbf16>) -> tensor<128x256xbf16>
+  // CHECK: linalg.matmul ins(%arg0, %arg1 : tensor<128x512xbf16>, tensor<512x256xbf16>) outs([[FILLED]] : tensor<128x256xbf16>) -> tensor<128x256xbf16>
+  %0 = onednn_graph.matmul %arg0, %arg1 : (tensor<128x512xbf16>, tensor<512x256xbf16>) -> tensor<128x256xbf16>
+  return %0 : tensor<128x256xbf16>
+}
+
+// CHECK-LABEL: @add
+func.func @add(%arg0: tensor<128x256xf32>, %arg1: tensor<128x256xf32>) -> tensor<128x256xf32> {
+  // CHECK: tensor.empty()
+  // CHECK: linalg.generic
+  // CHECK: arith.addf
+  %0 = onednn_graph.add %arg0, %arg1 : (tensor<128x256xf32>, tensor<128x256xf32>) -> tensor<128x256xf32>
```

Comment:
These can lower to named ops.

Reply:
I have a separate branch updating the lowering code and to suit our need after previous design discussions, this one only migrates current code in old repo. Since we are about to add lowering in a separate PR, I will squash them together in the lowering PR.